.travis.yml

```diff
@@ -18,8 +18,18 @@ before_script:
 script:
   - travis_retry docker build -t "$image" "$VARIANT"
   - ~/official-images/test/run.sh "$image"
+  - docker run -d --name blogtest -p 2368:2368 -e NODE_ENV=production "$image"
+  - until nc -z $(docker inspect --format='{{.NetworkSettings.IPAddress}}' blogtest) 2368; do
+      echo "waiting for ghostblog container...";
+      sleep 2;
+    done
+  - echo "GhostTest available..."
+  - curl http://localhost:2368 | grep "The professional publishing platform"
```

Comment:
See also the very similar https://github.com/docker-library/official-images/blob/f25156923caf093e61e8458b81ad827e9e101ae6/test/tests/ghost-basics/run.sh, which is run by `~/official-images/test/run.sh` above (and is run on every official-images PR for the `ghost` image as well).

Reply:
Cool. Thanks @tianon for the information about the ghost included in the official images. Thats great.